sdk/eventhub/event-hubs/README.md

```diff
@@ -357,9 +357,26 @@ The Event Hubs library depends on the [rhea-promise](https://github.com/amqp/rhe
 
 ### Enable logs
 
-You can set the following environment variable to get the debug logs when using this library.
+This library depends on the [@azure/logger](https://www.npmjs.com/package/@azure/logger) package
+to support debug logs.
 
-- Getting debug logs from the Event Hubs SDK
+You can set the `AZURE_LOG_LEVEL` environment variable to one of the following values to enable logging to `stderr`:
+
+- verbose
+- info
+- warning
+- error
+
+You can also set the log level programatically by importing the `@azure/logger` package and calling the
+`setLogLevel` function with one of the log level values.
+
+When setting a log level either programatically or via the `AZURE_LOG_LEVEL` environment variable,
+any logs that are written using a log level equal to or less than the one you choose will be emitted.
```

Comment:
An example here would be nice.

Reply:
Added a code sample!

Reply:
haha, I meant an example for the statement `logs that are written using a log level equal to or less than the one you choose will be emitted.`

Something like

> For example, when you set the log level to be `info`, the logs that are written for levels `warning` and `error` are also emitted.

On second thoughts, we could add a pointer to our logging policy instead?

I am not in favor of the example to programatically set the log level. The Readme of the logger package will have the relevant code snippets.

Reply:
Do you mean our TypeScript logging guidelines?